docs/src/reference/Optim.md

```diff
@@ -0,0 +1,6 @@
+# Optim 
+
+This folder contains 
```

Comment:
@vdebauche There is a problem here right ?

Reply:
That's because there is nothing for the moment. I will modify!